src/openms_gui/source/VISUAL/DataSelectionTabs.cpp

```diff
@@ -112,6 +112,13 @@ namespace OpenMS
     connect(this, &QTabWidget::tabBarDoubleClicked, this, &DataSelectionTabs::tabBarDoubleClicked);
   }
 
+  DataSelectionTabs::~DataSelectionTabs()
+  {
+    delete spectraview_controller_;
+    delete idview_controller_;
+    delete diatab_controller_;
```

Comment:
@cbielow can you check whether that is okay? I did some testing and these came up with `TOPPView_test` as leaking memory but can you have a look at what the ownership model is here and whether its safe to destroy them? or are these still needed later? in my tests it all ran fine

Reply:
+1
should be good.
I was wondering why I did not use a plain member, but its simply due to compilation time PIMPL.
We could simply make them members as well. Or switch to unique_ptr.